mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/identifierdialog/IdentifierDialogFragment.java

```diff
@@ -47,6 +48,9 @@
     @BindView(R.id.et_description)
     EditText et_description;
 
+    @BindView(R.id.btn_create_identifier)
+    Button btn_identifier;
```

Comment:
@AkshGautam Please use camel case naming convention and change the other fields name in this file too.

Reply:
@therajanmaurya  Done.